client/ayon_substancepainter/plugins/publish/extract_map_as_single_output.py

```diff
@@ -0,0 +1,96 @@
+import clique
+import os
+
+from ayon_core.pipeline import publish
+from ayon_core.lib import (
+    get_oiio_tool_args,
+    run_subprocess,
+)
+
+
+def convert_texture_maps_for_udim_export(staging_dir, image_outputs, has_udim=False):
```

Comment:
This function name is confusing - it doesn't 'convert' anything. It just gets the sequence filepaths. Please add a docstring, return type and improve function name,